### PR TITLE
Pull #1 of Sylabs PRs in preparation for v3.8.1

### PR DESCRIPTION
**Pull request by kmuriki**
This PR handles issues #6080 #6078 #6077 #6064 #6063 and #6062

### CODE REVIEW

File: e2e/instance/instance.go [hunk 1]
```diff
@@ -16,16 +16,26 @@ import (
...
+	t.Helper()
+
+	id, err := uuid.NewRandom()
+	if err != nil {
+		t.Fatal(err)
+	}
+	return id.String()
+}
+
+>>>>>>> 4f1cdf9... fix: partially replace UUID module
```
Comment (ikaneshiro) [inline e2e/instance/instance.go]: Left over merge text

Comment (kmuriki) [reply e2e/instance/instance.go]: Thanks @ikaneshiro Just fixed this.

**Comment by ikaneshiro** [comment]
@kmuriki It looks like the e2e tests are failing because https://github.com/sylabs/singularity/pull/22/commits/78f877863275a49226fbff6c1ea5330d457c7c5b has not been pulled in

**Comment by DrDaveD** [comment]
> @kmuriki It looks like the e2e tests are failing because [sylabs/singularity@78f8778](https://github.com/sylabs/singularity/commit/78f877863275a49226fbff6c1ea5330d457c7c5b) has not been pulled in

Yeah and many of the other ci tests are failing with the same error.

File: e2e/instance/instance.go [hunk 2]
```diff
@@ -309,7 +318,11 @@ func (c *ctx) applyCgroupsInstance(t *testing.T) {
 	}
 
 	// pick up a random name
+<<<<<<< HEAD
```
Comment (DrDaveD) [inline e2e/instance/instance.go]: I still see <<<< merge text here, and in regressions.go

**Comment by kmuriki** [comment]
Let me try pulling these PRs one at a time and resolve all these conflicts.

**Pull request closed without merging**